structure_tests/structure_test_release_cloudbuild.yaml

```diff
@@ -0,0 +1,12 @@
+steps:
```

Comment:
should we build `gcr.io/gcp-runtimes/structure-test-v2:$TAG_NAME` here as well, and then tag that to latest?

Reply:
I was assuming we'd use the retagger for that, we don't need to build it again since it's built every commit.

WDYT?

Reply:
Ah yeah, I guess it's just a little weird to have two separate jobs for the same "release". I definitely think we should use the retagger, but on the other hand one of the main things it gives us is the ability to easily rollback, which we'd be halfway bypassing by releasing a new binary through a `gsutil cp` here. Maybe we should have a single job that calls the retagger AND `gsutil cp`s the binary to the latest directory?

Reply:
Ah yeah, I think I'll drop the auto-aliasing of binaries.